src/browser/modules/Editor/Monaco.tsx

```diff
@@ -446,7 +455,7 @@ const Monaco = forwardRef<MonacoHandles, MonacoProps>(
       })
     }
 
-    return <MonacoStyleWrapper id={monacoId} />
+    return <MonacoStyleWrapper data-testid="monaco-editors" id={monacoId} />
```

Comment:
Since the components already have IDs that start with the same string, the mutual test id is not needed

Reply:
Cypress doesn't seem to have an easy way to do get with a regex, so I'm thinking this'd be worth keeping for that reason.